src/utils/Router.js

```diff
@@ -0,0 +1,43 @@
+import { Router } from 'express';
+import permissionsValidator from '../validators';
+
+class ExtendRouter {
+  constructor() {
+    this.innerRouter = new Router();
+    this.router = new Router();
+
+    this.setMethods(this.router);
+    this.innerRouter.stack = this.router.stack;
```

Comment:
Finally came up with how to do this Router. Trick in 2 routhers with common rout stack. Outher router with new "get, post..." methods get route and put it in common stack from where innerRouter calls own (original) methods for common stack.

What do you say, what I should use router or controller?

Reply:
Wow, 2 routers... looks like a code smell, this shouldn't be like this

Reply:
Okay, I understand...)